demos/card.scss

```diff
@@ -19,3 +19,167 @@
 @import "../packages/mdc-card/mdc-card";
 @import "../packages/mdc-checkbox/mdc-checkbox";
 @import "../packages/mdc-form-field/mdc-form-field";
+@import "../packages/mdc-icon-toggle/mdc-icon-toggle";
+@import "../packages/mdc-list/mdc-list";
+@import "../packages/mdc-ripple/mixins";
+
+.hero {
+  height: auto;
+  padding: 24px;
+}
+
+.demo-card-collection {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: center;
+  padding: 0 24px;
+}
+
+.demo-card-toggles {
+  margin: 24px;
+  padding: 24px;
+  text-align: center;
+}
+
+//
+// Card container
+//
+
+.demo-card {
+  width: 350px;
+  margin: 48px;
+}
+
+.demo-card--photo-album {
+  width: 200px;
+}
+
+//
+// Images
+//
+
+.demo-card__media {
+  background-image: url("/images/1-1.jpg");
+}
+
+.demo-card__media--16-9 {
+  background-image: url("/images/16-9.jpg");
+}
+
+.demo-card__media-content--with-title {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.demo-card__media-title {
+  padding: 8px 16px;
+  background-image: linear-gradient(to bottom, rgba(0,0,0,0) 0%,rgba(0,0,0,0.5) 100%);
+  color: white;
+}
+
+//
+// Focusable child elements
+//
+
+.demo-card__ripple-surface {
+  @include mdc-ripple-surface;
+  @include mdc-ripple-radius;
+  @include mdc-states;
+
+  &:not([data-mdc-ripple-is-unbounded]) {
```

Comment:
Would it make more sense to just apply these styles to `.demo-card-article` instead?

Reply:
Potentially, but if we ever decide to add another card example that uses
`demo-card__ripple-surface` on a different element, it won't get the benefit of the
IE/Edge workaround (`position: relative`).